### PR TITLE
Add manufacturer data blocklist

### PR DESCRIPTION
**Pull request by chengweih001**
Hi WebBluetoothCG/registries maintainer,

This PR creates a manufacturer data block list according to [crbug.com/1163207](https://crbug.com/1163207). Could you please review it?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,9 +12,16 @@ holds a list of 128-bit GATT UUIDs that
 websites using the Web Bluetooth API are forbidden from accessing.
 This includes all of Services, Characteristics, and Descriptors.
 
-This file contains comments in lines starting with `#`.
-Non-comment lines contain a [valid UUID](https://webbluetoothcg.github.io/web-bluetooth/#valid-uuid)
+[manufacturer_data_blocklist.txt](/manufacturer_data_blocklist.txt)
+holds a list of manufacturer data that websites using the Web Bluetooth API are forbidden from accessing.
+Each entry of the list is consist of company identifier and data prefix that should be blocked.
```
Comment (scheib) [inline README.md]: "Each entry of the list consists of a company ..."

File: README.md [hunk 2]
```diff
@@ -12,9 +12,16 @@ holds a list of 128-bit GATT UUIDs that
...
+
+These files contain comments in lines starting with `#`.
+Non-comment lines contain one of these:
+  * A [valid UUID](https://webbluetoothcg.github.io/web-bluetooth/#valid-uuid)
 followed optionally by a space and one of the tokens "`exclude-reads`" or "`exclude-writes`".
+  *  The word `manufacturer` followed by a space and a group of hexadecimal digits (at most four hex digits)
+  that represents a company identifier. A space after that, is `advdata-` followed by a data prefix with mask.
```
Comment (scheib) [inline README.md]: Can we cite a registry of company identifiers?

Comment (scheib) [inline README.md]: The advertising data syntax would benefit from more description.  Say in words "Advertising data" so people don't have to guess or parse advdata, and describe the way the data and mask are written?

File: README.md (hunk 1 above)
Comment (chengweih001) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (chengweih001) [reply README.md]: Done.

Comment (chengweih001) [reply README.md]: Added more detail to explain the data prefix format and how to match an advertising data.

File: README.md [hunk 3]
```diff
@@ -14,14 +14,20 @@ This includes all of Services, Characteristics, and Descriptors.
...
+Each entry of the list consists of a company identifier and data prefix that should be blocked.
 
 These files contain comments in lines starting with `#`.
 Non-comment lines contain one of these:
   * A [valid UUID](https://webbluetoothcg.github.io/web-bluetooth/#valid-uuid)
 followed optionally by a space and one of the tokens "`exclude-reads`" or "`exclude-writes`".
-  *  The word `manufacturer` followed by a space and a group of hexadecimal digits (at most four hex digits)
-  that represents a company identifier. A space after that, is `advdata-` followed by a data prefix with mask.
+  *  The word `manufacturer` followed by a space and a group of hexadecimal digits (at most four
+  hexadecimal digits) that represents a company identifier, whcih can be found on the
```
Comment (scheib) [inline README.md]: small: "whcih" -> "which"

File: README.md [hunk 4]
```diff
@@ -14,14 +14,20 @@ This includes all of Services, Characteristics, and Descriptors.
...
+  [Bluetooth Assigned Numbers website](https://www.bluetooth.com/specifications/assigned-numbers).
+  A space after that is an advertising data prefix in the format `advdata-<data>/<mask>`, where
+  `data` and `mask` are strings of hexadecimal digits of the same length. The byte order of the
+  data prefix is from left to right. An advertising data matches the data prefix if each byte of
```
Comment (scheib) [inline README.md]: "left to right" isn't specific enough.  Use "The left most character is the most significant byte" (or flipped if that's what you're saying accurate). Or, if little/big-endian makes more sense, use that (though I find that less specific).

**Comment by scheib** [comment]
Small suggestions remain

**Comment by scheib** [comment]
Some additional comments

File: README.md (hunk 3 above)
Comment (chengweih001) [reply README.md]: Done. Fixed in https://github.com/WebBluetoothCG/registries/pull/31/commits/554eff0addeaa1fdf326aa30ad3da1d27b7d544d.

File: README.md (hunk 4 above)
Comment (chengweih001) [reply README.md]: I updated to use little-endian and add an example. https://github.com/WebBluetoothCG/registries/pull/31/commits/44c8810536fe6ef17dfaffacd53a779001bfff24

File: README.md [hunk 5]
```diff
@@ -12,9 +12,23 @@ holds a list of 128-bit GATT UUIDs that
...
+
+These files contain comments in lines starting with `#`.
+Non-comment lines contain one of these:
+  * A [valid UUID](https://webbluetoothcg.github.io/web-bluetooth/#valid-uuid)
 followed optionally by a space and one of the tokens "`exclude-reads`" or "`exclude-writes`".
+  *  The word `manufacturer` followed by a space and a group of hexadecimal digits (at most four
+  hexadecimal digits) that represents a company identifier, which can be found on the
+  [Bluetooth Assigned Numbers website](https://www.bluetooth.com/specifications/assigned-numbers).
+  A space after that is an advertising data prefix in the format `advdata-<data>/<mask>`, where
+  `data` and `mask` are strings of hexadecimal digits of the same length. The byte order of the data
```
Comment (scheib) [inline README.md]: small change please on wording, such as:


... strings of bytes in hexadecimal, e.g. `advdata-001100/00FF00`.

A data prefix matches an advertising data if each byte of the prefix is equal to the corresponding byte of the advertising data after both are ANDed with the corresponding byte of the mask.

E.g. the example earlier would match data such as 
`11` `11` `11`, or 
`00` `11` `22`, etc.

**Review by scheib — APPROVED** [review]

**Pull request merged**